10.9-libcxx/stable/main/finkinfo/languages/libyaml.info

```diff
@@ -1,31 +1,39 @@
 Package: libyaml
-Version: 0.1.6
-Revision: 1
+Version: 0.2.1
+Revision: 2
 Depends: %N-shlibs (= %v-%r)
 BuildDependsOnly: true
 Source: http://pyyaml.org/download/%n/yaml-%v.tar.gz
-Source-MD5: 5fe00cda18ca5daeb43762b80c38e06e
+Source-MD5: 72724b9736923c517e5a8fc6757ef03d
+PatchFile: %n.patch
+PatchFile-MD5: 2fdfd0f847760e0c46a6104dfedc4fd9
+
 PatchScript: <<
-	# Patch configure to not link like Puma on Yosemite
-	perl -pi.bak -e 's|10\.\[012\]\*|10.[012][,.]*|g' configure
+  # Patch configure to not link like Puma on Yosemite
+  perl -pi.bak -e 's|10\.\[012\]\*|10.[012][,.]*|g' configure
+
+  # Patch from upstream to fix test with yaml-py
```

Comment:
Where did the version come from (give a URL)?  I think just replying to this comment is enough documentation.

Reply:
I deduced it from the report  yaml/libyaml#122, mentioned above. Corresponding comment added to libyaml.info.